components/Carousel.js

```diff
@@ -22,9 +23,7 @@ export default class Carousel extends React.Component {
 						href={item} target="_blank"
 						rel="noreferrer noopener"
 						key={i}>
-							<div
-							style={{backgroundImage: 'url('+ process.env.PUBLIC_URL + item + ')'}}>
-							</div>
+							<Image src={item} width={IMAGE_WIDTH} height={IMAGE_WIDTH}/>
```

Comment:
Yeah, so my reasoning here is that the background image approach itself is a hack - each `div` really semantically should be an image, and without this change (in general) the Carousel would be inaccessible.

I did merge that above change with *also* transitioning it to `<Image />`, which may have jumped the gun. Didn't see any issues with this on prod, but always down to switch back!

Reply:
Makes sense, re: accessibility.

If no problems seen on prod with `next/image` I think it's actually a great change. The carousel is a prime place for lazy loading since it's located below the fold and is image dense